ffi/src/log.rs

```diff
@@ -0,0 +1,187 @@
+use std::{
+    fmt::{Debug, Formatter},
+    mem::ManuallyDrop,
+    sync::Once,
+};
+
+pub use log::LevelFilter;
+use log::{Level, Log, Metadata, Record};
+use simple_logger::SimpleLogger;
+
+/// Configure logging for the library.
+#[no_mangle]
+pub extern "C" fn set_log_level(level: LevelFilter) -> i32 {
```

Comment:
Go still has to call this only once to set the logging level and then it cannot be changed?

Reply:
This fn was configuring a logger printing to stdout. Removed it in favor of `set_logging_callback()`.